cmd/server-startup-msg.go

```diff
@@ -67,6 +71,31 @@ func printStartupMessage(apiEndPoints []string) {
 	}
 }
 
+// strip api endpoints list with standard ports such as 
+// port "80" and "443" before displaying on the startup
+// banner.  Returns a new list of API endpoints.
+func stripStandardPorts(apiEndpoints []string) (newAPIEndpoints []string) {
+	newAPIEndpoints = make([]string, len(apiEndpoints))
+	// Check all API endpoints for standard ports and strip them.
+	for i, apiEndpoint := range apiEndpoints {
+		url, err := url.Parse(apiEndpoint)
+		if err != nil {
+			newAPIEndpoints[i] = apiEndpoint
+			continue
+		}
+		host, port, _ := net.SplitHostPort(url.Host)
+		// For standard HTTP(s) ports such as "80" and "443"
+		// apiEndpoints should only be host without port.
+		if port == "80" || port == "443" {
```

Comment:
Can you enforce scheme to http if port is 80 and https if port is 443, even when no tls certificates are provided. I know this is controversial but let's imagine if the user enters --address ":443", in that case :443 will be stripped and the resulted url will still be http://host and not https://host.